charts/newrelic-infrastructure-beta/templates/NOTES.txt

```diff
@@ -0,0 +1,34 @@
+Your deployment of the New Relic Infrastructure agent is complete. You can check on the progress of this by running the following command:
+
+    kubectl get daemonset -o wide -w --namespace {{ .Release.Namespace }} {{ template "newrelic.fullname" . }}
+
+{{- if .Values.image }}
+{{- if and (eq .Values.enableWindows true) (.Values.image.windowsTag) }}
+
+####       .Values.image.windowsTag for [newrelic-infrastructure] is deprecated      ####
+####  it will be removed in the future, please consider to update the configuration  ####
+
+{{- end -}}
+{{- if and (include "newrelic.fargate" .) .Values.nodeAffinity }}
+###########
+# WARNING #
+###########
+
+You have specified both an EKS Fargate environment (global.fargate) and custom
+nodeAffinity rules, so we couldn't automatically exclude the daemonSet from
+Fargate nodes. In order for the integration to work, you MUST manually exclude
+the daemonSet from Fargate nodes.
+
+Please make sure your custom .Values.nodeAffinity achieve the same effect as:
+
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: eks.amazonaws.com/compute-type
+              operator: NotIn
+              values:
+                - fargate
+{{- end }}
+{{- end }}
```

Comment:
Unrelated to this PR but there is likely a bug in here: The first if (`if .Values.image`) should not cover the fargate warning part.